core-bundle/contao/templates/twig/content_element/downloads.html.twig

```diff
@@ -13,3 +13,12 @@
 {% block list_item %}
     {% with {download: item} %}{{ block('download_component') }}{% endwith %}
 {% endblock %}
+
+{# Add metadata #}
+{% block schema_org %}
+    {% for download in downloads %}
+        {% if download.file is defined and download.file.schemaOrgData|default(false) %}
+            {% do add_schema_org(download.file.schemaOrgData) %}
+        {% endif %}
```

Comment:
The default filter also handles undefined parameters, you could omit the first check.

Btw, if we would also allow `null` or empty arrays in `SchemaOrgRuntime#add()`, we could even do this (which is more readable to me):
```suggestion
        {% do add_schema_org(download.file.schemaOrgData|default) %}
```

Reply:
Done in f63447ca402f50b57f7d881b1aafc22f8ab4d4ff.